catalyst_voices/lib/routes/guards/unlocked_keychain_guard_mixin.dart

```diff
@@ -0,0 +1,22 @@
+import 'dart:async';
+
+import 'package:catalyst_voices/routes/routing/spaces_route.dart';
+import 'package:catalyst_voices_blocs/catalyst_voices_blocs.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:go_router/go_router.dart';
+
+/// A guard mixin which redirects to the discovery page
+/// if the keychain is locked.
+mixin UnlockedKeychainGuardMixin on GoRouteData {
```

Comment:
Such general mixin will give us problems when there will be need for more then one guard.

Think about implementing `SesssionUnlockedGuard` like `MilestoneGuard` and convert this mixin to something that can handle multiple `RouteGuard`s + given Route itself option for overriding `redirect`

Reply:
ok, I see. Will think about it